src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/controllers/CcAttachSupplementaryEvidenceTest.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.controllers
+
+import io.restassured.RestAssured
+import io.restassured.RestAssured.given
+import io.restassured.builder.RequestSpecBuilder
+import io.restassured.http.ContentType.JSON
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.springframework.boot.web.server.LocalServerPort
+import org.springframework.test.context.junit.jupiter.SpringExtension
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.PortWaiter.waitFor
+import uk.gov.hmcts.reform.bulkscan.orchestrator.controllers.config.IntegrationTest
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest
+
+@ExtendWith(SpringExtension::class)
+@IntegrationTest
+class CcdAttachSupplementaryEvidenceTest {
```

Comment:
Jira story says 'Linking exception record to a case'. Is this the same???

Reply:
yup .. its the same thing.
It really should be case with scanned documents I believe.

Reply:
how about? `AttachExceptionRecordToExistingCase`

Reply:
sure ... can do